src/RealComponents.ml

```diff
@@ -0,0 +1,154 @@
+open Base
+
+open Expr
+open Utils
+
+let value_of : Value.t -> float =
+  function [@warning "-8"]
+  | Real x -> x
+  | String "" -> 0.
+
```

Comment:
is there a reason why we would get empty string?

Reply:
Ah I copied that over from the ExcelSynth version of LoopInvGen I was referring to, but probably doesn't apply for this the same way. Should I take it out?

Reply:
yes, I think you can remove this and use the value type in the function. thanks!

Reply:
okay should be fixed now!